Makefile

```diff
@@ -23,7 +23,7 @@ _build:
 	go build -v -ldflags="-X github.com/Azure/acs-engine/cmd.BuildSHA=${VERSION} -X github.com/Azure/acs-engine/cmd.BuildTime=${BUILD}"
 	cd test/acs-engine-test; go build -v
 
-build: prereqs _build
+build: _build
```

Comment:
Might as well remove `_build` and replace with `build`. I had kept `build` so that a naive `make` or `make build` would work out of the box. If `build` doesn't automatically invoke `glide`, then the user has to know to run an additional step. But I'm fine with it either way.